samples/dotnet_core_managed_gateway/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "dotnet_core_managed_gateway": {
+      "commandName": "Project",
+      "commandLineArgs": "C:\\repos\\azure-iot-gateway-sdk\\samples\\dotnet_core_managed_gateway\\dotnet_core_managed_gateway_win.json"
```

Comment:
You've got a hard-coded path here. Also, shouldn't the path have forward slashes instead of escaped back slashes (to work on Linux)?